scss/_nav.scss

```diff
@@ -112,6 +113,12 @@
     background: none;
     border: 0;
     @include border-radius(var(--#{$prefix}nav-pills-border-radius));
+
+    &:disabled {
```

Comment:
Should this have both selectors, class and attribute, or just this one?

Reply:
just `:disabled` because it overrides styles specific only to `:disabled`